elf/elf.go

```diff
@@ -257,7 +322,21 @@ func elfReadMaps(file *elf.File) (map[string]*Map, error) {
 			mapCount := len(data) / C.sizeof_struct_bpf_map_def
```

Comment:
Could you check if `len(data)` is really a multiple of `C.sizeof_struct_bpf_map_def`? If another user of the gobpf library has ELF files with the definition of `struct bpf_map_def` before this PR and use gobpf code after this PR, this would help them to detect the issue.